### DISCUSSION TITLE
what actually extended request params RequestInit.next.tags do?

### DISCUSSION THREAD

**Discussion by dante01yoon** [vercel/next.js#51168] [App Router]
I see only `revalidate` notes in docs. 

what actually tags do when use fetch?

**Comment by devjiwonchoi** [comment]
It is used to `revalidate` the `cache tags` in order to "update your cached data without waiting for a revalidation period to expire".

> tag: A string representing the cache tag associated with the data you want to revalidate.

You can use either `revalidateTag` or `fetch`.

```js
revalidateTag(tag: string): void;

// or

fetch(url, { next: { tags: [...] } })
```

> You can add tags to fetch as follows:
> - fetch(url, { next: { tags: [...] } });

## Reference
- [`revalidateTag` Parameters](https://nextjs.org/docs/app/api-reference/functions/revalidateTag#parameters)
